recipe/bld.bat

```diff
@@ -15,7 +15,7 @@ set "CFLAGS=%CFLAGS% -DU_HAVE_STDINT_H=1"
 
 bash runConfigureICU MSYS/MSVC --prefix=%LIBRARY_PREFIX% --enable-static
 if errorlevel 1 exit 1
-make
+make -j%NUMBER_OF_PROCESSORS%
```

Comment:
Better to avoid -j.  CIs do not always report it reliably.  Instead,  prefer to set MAKEFLAGS and let conda build pass that along.